functions/index.js

```diff
@@ -0,0 +1,24 @@
+const functions = require('firebase-functions');
+const fetch = require('node-fetch');
+const HONESTY_STORE_LOGO = 'https://honesty.store/assets/android/icon@MDPI.png';
+const BOT_USERNAME = 'honesty.store';
+
+exports.sendSlackMessage = functions.https.onCall(data => {
+  const token = functions.config().react_app.slack_token;
```

Comment:
Can we drop the `react_app` prefix now they're no longer used in the client codebase?